templates/taxi/car_detail.html

```diff
@@ -2,21 +2,43 @@
 
 {% block content %}
   <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    Model: {{ car.model }}
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+
+  <div class="row">
+    <div class="col-5">
+      <h3>Drivers</h3>
+    </div>
+    <div class="col-7">
+      <form action="{% url 'taxi:assign-driver' car.pk %}" method="post">
```

Comment:
Ensure that the URL name 'taxi:assign-driver' used in the form action matches the URL configuration in your Django project. If the URL pattern is named differently, update this to prevent form submission errors.